src/index.html

```diff
@@ -10,8 +10,8 @@ <h1 class="hero__title">
       <h3 class="hero__date">SINCE 1987</h3>
       <hr class="hero__divider">
       <p class="hero__tagline">Premium Quality Industrial Safety Garments</p>
-      <div>
-          <a class="button button-yellow" href="/products">OUR PRODUCTS</a>
+      <div class="hero__button__row">
```

Comment:
Comment on previous commit to remove this.
Note that this is not BEM syntax.
[CSS Wizardry](https://csswizardry.com/2013/01/mindbemding-getting-your-head-round-bem-syntax/)
[Yandex](http://getbem.com/introduction/)